src/ja/server/diagrams/schedulerLowLevel.puml

```diff
@@ -0,0 +1,129 @@
+@startuml
+
+box "Central Server" #Lightblue
+    participant ":QueuingActivity"
+    participant ":Scheduler"
+    participant "job:Job"
+    participant ":RunningJobEntry"
+    participant ":DataBaseHandler"
+    participant ":Dispatcher"
+end box
+
+activate "job:Job"
+-> ":QueuingActivity":
+activate ":QueuingActivity"
+
+":QueuingActivity" -> ":Scheduler" ++: queueJob(job:Job)
```

Comment:
Scheduler is supposed to retrieve list of Jobs from the Database.
When a Job is added that is merely a trigger for the Scheduler to retrieve a list of Jobs.